frameworks/kafka/src/main/dist/krb5.conf.mustache

```diff
@@ -0,0 +1,17 @@
+[libdefaults]
```

Comment:
@NimaVaziri @benclarkwood how should a minimal `krb5.conf` file look. We at least need the KDC host.

Reply:
I'll take a look at this today. My thinking is that we can either
-  include all the settings but template them since this file will be an artifact of the package and it's up to the operator to configure this file.
- or (as an anti-pattern) have this file be a resource that's fetched.

Reply:
@imaxxs This is the config file.

Reply:
@NimaVaziri @imaxxs @benclarkwood I have removed most of the settings here, only specifying the default realm and the kdc host for that realm. Here, I would like to get more input from users as to what settings are typically required.

Reply:
Looks good for MVP/first pass.

We can interact with JPMC again, and also audit what sorts of questions people are asking on the interwebs about setting up Kerberos.